include/cantera/kinetics/LinearBurkeRate.h

```diff
@@ -0,0 +1,154 @@
+//! @file LinearBurkeRate.h
+// This file is part of Cantera. See License.txt in the top-level directory or
+// at https://cantera.org/license.txt for license and copyright information.
+
+#ifndef CT_LINEARBURKERATE_H
+#define CT_LINEARBURKERATE_H
+#include "cantera/kinetics/Arrhenius.h"
+#include <boost/variant.hpp>
+#include "cantera/kinetics/Falloff.h"
+#include "cantera/kinetics/ChebyshevRate.h"
+#include "cantera/kinetics/PlogRate.h"
+
+namespace Cantera
+{
+
+//! Data container holding shared data specific to LinearBurkeRate
+/**
+ * The data container `LinearBurkeData` holds precalculated data common to
+ * all `LinearBurkeRate` objects.
+ */
+struct LinearBurkeData : public ReactionData
+{
+    LinearBurkeData();
+
+    void update(double T, double P) override
+    {
+        ReactionData::update(T);
+        pressure = P;
+        logP = std::log(P);
+    }
+
+    bool update(const ThermoPhase& phase, const Kinetics& kin) override;
+
+    using ReactionData::update;
+
+    //! Perturb pressure of data container
+    /**
+     * The method is used for the evaluation of numerical derivatives.
+     * @param  deltaP  relative pressure perturbation
+     */
+    void perturbPressure(double deltaP);
+
+    void restore() override;
+
+    virtual void resize(size_t nSpecies, size_t nReactions, size_t nPhases) override
+    {
+        moleFractions.resize(nSpecies, NAN);
+        ready = true;
+    }
+
+    void invalidateCache() override
+    {
+        ReactionData::invalidateCache();
+        pressure = NAN;
+    }
+
+    double pressure = NAN; //!< Pressure
+    double logP = 0.0; //!< Logarithm of pressure
+    bool ready = false; //!< Boolean indicating whether vectors are accessible
+    vector<double> moleFractions;
+    int mf_number;
+
+protected:
+    double m_pressure_buf = -1.0;
+};
+
+
+//! Pressure-dependent and composition-dependent reaction rate calculated according to
+//! the reduced-pressure linear mixture rule (LMR-R) developed at Columbia University.
+//! @cite singal2024
```

Comment:
In Doxygen, the `cite` command just inserts the numeric reference, so the "Author et al" context needs to be written out manually. Also, the first sentence (identified by the first period, or a blank line) is treated specially by Doxygen as the "brief" documentation for a class / method.

Finally, it's good to link back to the reference documentation like this (a bit tricky since we're mixing documentation systems).

```suggestion
//! Pressure-dependent and composition-dependent reaction rate calculated according to
//! the reduced-pressure linear mixture rule (LMR-R)
//!
//! This parameterization is described by Singal et al. @cite singal2024 and in the
//! [science reference](../reference/kinetics/rate-constants.html#linear-burke-rate-expressions)
//! documentation.
```